src/main/modules/SnippetsManager.js

```diff
@@ -188,12 +188,31 @@ class SnippetsManager {
     }
 
     replace(value) {
-        const clipboardContent = this.clipboard.readText()
+        try {
+            // 1. store old clipboard data
```

Comment:
I don't think comments are necessary nor are adding any value. Let's write code that doesn't require comments ;)

Reply:
Sure, as you wish